lib/android.js

```diff
@@ -70,7 +70,8 @@ class OptionalAppBundleCheck extends DoctorCheck {
   }
 
   async fix () { // eslint-disable-line require-await
-    return 'bundletool.jar is used to handle Android App Bundle. Please read http://appium.io/docs/en/writing-running-appium/android/android-appbundle/ to install it';
+    return 'bundletool.jar is used to handle Android App Bundle. Please read http://appium.io/docs/en/writing-running-appium/android/android-appbundle/ to install it' +
+      `${system.isWindows() ? '. Please consider adding ".JAR" in your PATHEXT in order to find it' : ''}`;
```

Comment:
Also consider adding the ".jar" extension into your PATHEXT environment variable in order to fix the problem for Windows